ckan/controllers/user.py

```diff
@@ -390,12 +430,20 @@ def _save_new(self, context):
             set_repoze_user(data_dict['name'])
 
             if request.params:
-                self.email_new_user_details_using_mailgun_and_log_in_bpam(request.params)
+                if request.params['project_of_interest'] == 'Australian Microbiome':
```

Comment:
maybe let's put an array somewhere like`'AUTOREGISTER_PROJECTS` so that we can easily flip this on for other projects later

Reply:
Fixed.